diff -urN Alt-Lexical-Var-ButSupportModernPerl-0.001/t/scalar_const.t Alt-Lexical-Var-ButSupportModernPerl-patched/t/scalar_const.t
--- Alt-Lexical-Var-ButSupportModernPerl-0.001/t/scalar_const.t	2022-09-16 13:58:23.000000000 +0100
+++ Alt-Lexical-Var-ButSupportModernPerl-patched/t/scalar_const.t	2022-10-17 10:34:05.879023812 +0100
@@ -58,7 +58,7 @@
 	use Lexical::Var '$foo' => \(my $x=undef);
 	$foo = 456;
 };
-like $@, qr/\ADied /;
+like $@, qr/\A(Died|Can't modify) /;
 
 # test that non-constant undef $foo does not participate in constant folding
 eval q{
@@ -66,7 +66,7 @@
 	use Lexical::Var '$foo' => \(my $x=undef);
 	!$foo = 456;
 };
-like $@, qr/\ACan't modify not /;
+like $@, qr/\ACan't modify /;
 
 is eval q{
 	use Lexical::Var '$foo' => \undef;
